_pages/testing.md

```diff
@@ -9,14 +9,16 @@ redirect_from:
+Login.gov provides an open sandbox environment to create and test integrations between login.gov and you applications. In the sandbox environment, we provide a dashboard where you can manage your test applications.
 
-1. Request that accounts your emails get added to the dashboard application in the test sandbox environment. You can request [access here](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e). The login.gov team will create accounts for you and your team members. Once you have an account, head over to the dashboard at [dashboard.int.identitysandbox.gov](https://dashboard.int.identitysandbox.gov).
-2. In the upper-right corner, click **Log in**. You'll be prompted to sign in or create an account with the test login.gov IdP in the agency integration environment (hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov)). **Please note that if you already have a login.gov account that this is a test environment that is not linked to your production account.**
-3. Once logged in, you'll be sent to the "My apps" page, which will list all the test applications you and your team have created.
-4. Click **Create a new test app** and fill out the form to register a new application with the login.gov IdP in the test sandbox environment.
-5. Start testing! If you need to trouble shoot, please send us an email and we can onboard you to our partner support slack channel and the login.gov team will help you along the way.
-6. When you're ready to go to production, please contact our team at partners@login.gov and notify us. We'll manage your application's promotion to production.
+Anybody with an email address ending in .gov or .mil can create an account in the sandbox environment. If your email address does not end in .gov or .mil ask your federal partner to create a sandbox team and add you to that team.
+
+1. In the upper-right corner, click **Log in**. You'll be prompted to sign in or create an account with the test login.gov IdP in the agency integration environment (hosted at [idp.int.identitysandbox.gov](https://idp.int.identitysandbox.gov)). **Please note that if you already have a login.gov account that this is a test environment that is not linked to your production account.**
+2. Once logged in you'll be asked to create a team and add users.
+2. After creating your team visit the "Apps" tab. This page will list all the test applications you and your team have created.
+3. Click **Create a new test app** and fill out the form to register a new application with the login.gov IdP in the test sandbox environment.
+4. Start testing! If you need to trouble shoot, please send us an email and we can onboard you to our partner support slack channel and the login.gov team will help you along the way.
```

Comment:
s trouble shoot/troubleshoot
s slack/Slack

- What email address will they be sending their questions to? Let's make `email` a link to that email address.

Reply:
I believe it should be 'partners@login.gov'. I'll update.